### PR TITLE
Janeiro Point Wonder 🏛

### PR DESCRIPTION
**Pull request by DukeJeans**
## Changelog:
- Added in unique wonder, **Janeiro's Point**, to Booty Bay province.
  - Functions as a statue great work

### For Keepers:
Please test to ensure all images and localisation is unique and valid. The wonder should not default to any sort of vanilla or missing localisation.

### CODE REVIEW

File: localisation/00_wc_wonders.csv [hunk 1]
```diff
@@ -100,3 +100,7 @@ wonder_shrine_storm_construction_desc;;;;;;;;;;;;;;x
 wonder_ulduar;Ulduar;;Ulduar;;;;;;;;;;;x
 wonder_ulduar_desc;Ulduar is the biggest titan-forged complex on Azeroth. Situated in the Storm Peaks, it notably encompasses the Halls of Stone and the Halls of Lightning.;;Ulduar ist der gr��te von Titanen geschmiedete Komplex auf Azeroth. Er befindet sich in den Sturmspitzen und umfasst insbesondere die Hallen der Steine und die Hallen des Blitzes.;;;;;;;;;;;x
 wonder_ulduar_construction_desc;;;;;;;;;;;;;;x
+# Janeiro Point;;;;;;;;;;;;;;
+wonder_janeiro_point;Janeiro Point;Cap Janeiro;;;;;;;;;;;;x
+wonder_janeiro_point_desc;Janeiro Point is a small island just off the coast of Booty Bay. Upon it stands a statue of a goblin holding his arms stretched in welcoming contentment.;Le Cap Janeiro est une petite �le sur la c�te de la Baie du Butin. Sur celle-ci se tient une statue d'un gobelin aux bras tendus qui a �t� construite pour servir de phare � tous les voyageurs.;;;;;;;;;;;;x
```
Comment (arithon) [inline localisation/00_wc_wonders.csv]: shouldn't it be Janeiro's Point?

Comment (DukeJeans) [reply localisation/00_wc_wonders.csv]: Good catch, thanks

File: common/wonders/wc_wonders_historical.txt [hunk 2]
```diff
@@ -621,3 +620,105 @@ wonder_ulduar = {
 		}
 	}
 }
+
+#Janeiro Point
+wonder_janeiro_point = {
```
Comment (zumbak04) [inline common/wonders/wc_wonders_historical.txt:625]: The upgrades look a bit weird to me.
Like if it's a statue, why not add the statue upgrades?

File: common/wonders/wc_wonders_historical.txt [hunk 3]
```diff
@@ -621,3 +620,105 @@ wonder_ulduar = {
...
+	allow_construction = no
+	picture = GFX_wonder_strip_janeiro_point
```
Comment (zumbak04) [inline common/wonders/wc_wonders_historical.txt:627]: There are better pictures of the Booty Bay.

<details>
<summary>Click to expand</summary>





</details>

File: common/wonders/wc_wonders_historical.txt [hunk 4]
```diff
@@ -621,3 +620,105 @@ wonder_ulduar = {
...
+	potential = {
+	}
+	active = {
+	}
+	flags = {
+		is_historical
+		cost_medium
```
Comment (zumbak04) [inline common/wonders/wc_wonders_historical.txt:634]: Why it's medium? The first stage of `wonder_statue_ruler` costs `1500` and it's considered `cost_low`.

File: common/wonders/wc_wonders_historical.txt [hunk 5]
```diff
@@ -621,3 +620,105 @@ wonder_ulduar = {
...
+
+#Janeiro Point
+wonder_janeiro_point = {
+	allow_construction = no
+	picture = GFX_wonder_strip_janeiro_point
+	potential = {
+	}
+	active = {
+	}
+	flags = {
```
Comment (zumbak04) [inline common/wonders/wc_wonders_historical.txt:632]: It doesn't have the statue flags.

File: common/wonder_upgrades/wc_wonders_historical_upgrades.txt [hunk 6]
```diff
@@ -66,4 +64,23 @@ upgrade_crucible_of_storms = {
...
+	}
+	build_time = 24
+	gold_cost_flat = 400
+	gold_cost_ticking = 1
+	local_modifier = {
+		galleys_perc = -0.10
+	}
+	owner_modifier = {
+	}
 }
```
Comment (MemeWizard) [inline common/wonder_upgrades/wc_wonders_historical_upgrades.txt:86]: > Mok'rash, a sea giant, calls this island his home, and he is responsible for quite a few sunken ships wanting to pull into the bay.

I don't think it's good idea to add this as an upgrade

1 - There is no way of removing it from wonder, except loot the wonder.
2 - After looting the wonder, anyone can build this upgrade again.

Comment (arithon) [reply common/wonder_upgrades/wc_wonders_historical_upgrades.txt:86]: maybe replace by Mok'rash's treasury?

Comment (DukeJeans) [reply common/wonder_upgrades/wc_wonders_historical_upgrades.txt:86]: Do you know of a better alternative, maybe a province modifier? I just like Mok'rash for the easter egg/flavor of the province. He's not ultimately necessary for this branch

Comment (zumbak04) [reply common/wonder_upgrades/wc_wonders_historical_upgrades.txt:86]: > 1 - There is no way of removing it from wonder, except loot the wonder.

@MemeWizar, isn't it impossible even via looting?

Comment (DukeJeans) [reply common/wonder_upgrades/wc_wonders_historical_upgrades.txt]: Mok'rash killed by looting in test, and player is able to rebuild, `allow_construction` is non-applicable to upgrades, so Mok'rash is removed

**Pull request merged**